api/handlers/listhandler.py

```diff
@@ -547,6 +547,12 @@ def delete(self, cont_name, list_name, **kwargs):
         permchecker, storage, _, _, keycheck = self._initialize_request(cont_name, list_name, _id, query_params=kwargs)
 
         permchecker(noop)('DELETE', _id=_id, query_params=kwargs)
+
+        if cont_name == 'acquisitions':
+            analysis_ids = containerutil.get_analyses(_id, kwargs['name'])
```

Comment:
This wouldn't allow a user to delete a file in an acquisition if a different file in that acquisition was used as an input to an analysis. Could we modify `containerutil.get_analyses()` to take an optional filename and match on that if provided?

Reply:
`get_analyses()` (now `get_referring_analyses()`) worked exactly like that. I now added a test for it.

Reply:
Ah my apologies I misread the function. Thanks for adding a test.